homeassistant/components/nuki/__init__.py

```diff
@@ -178,3 +126,99 @@ def __init__(
         """Pass coordinator to CoordinatorEntity."""
         super().__init__(coordinator)
         self._nuki_device = nuki_device
+
+    @property
+    def name(self) -> str:
+        """Return the name of the lock."""
+        return self._nuki_device.name
```

Comment:
Let's use `_attr_*` at `__init__`

Reply:
moved to `_attr_name` in `__init__`

Reply:
I don't see where `_attr_name` is used.

Reply:
it was used in the device registration, but I changed it as it was interfering with the entity naming.

Reply:
`_attr_name` for the entities will be implemented in https://github.com/home-assistant/core/pull/80021

Reply:
Ok. For the future, it's good to avoid removing features until they are replaced with another implementation.

I'll tag #80021 for 2023.2.0 to avoid loosing the entity name.